nrfutil/sandbox.ts

```diff
@@ -26,10 +31,15 @@ import {
 } from './sandboxTypes';
 
 const parseJsonBuffers = <T>(data: Buffer): T[] | undefined => {
+    // bcz if data is undefined data.toString().trim() will throw an exception
+    if (!data) return undefined;
```

Comment:
Like above, do not add this sanity check here. The clients of this function already take care of `data` always being defined.